src/s11/handlers/s11_DS_resp_handler.c

```diff
@@ -46,23 +46,23 @@ s11_DS_resp_handler(MsgBuffer* message, GtpV2MessageHeader* hdr, uint32_t sgw_ip
 	 * */
 	if(hdr->teid)
     {
-        dsr_info.ue_idx = hdr->teid;
+        dsr_info.s11_mme_cp_teid =  hdr->teid;
```

Comment:
Why we are calling it s11_mme_cp_teid ? is this just variable name change or something more ?

Reply:
Its just the variable name change.